docs/developer-guide/coordinate-systems.md

```diff
@@ -19,42 +17,69 @@ new PointCloudLayer({
-| `COORDINATE_SYSTEM.METER_OFFSETS` *   | [Δx, Δy, Δz]   | meters | Positions are given in meter offsets from a reference geo-location that is specified separately (`coordinateOrigin`). The `x` axis points map east, the `y` axis points map north, and `z` points up. |
-| `COORDINATE_SYSTEM.LNGLAT_OFFSETS`    | [Δlongitude, Δlatitude, Δaltitude]   | meters | Positions are given in meter offsets from a reference geo-location that is specified separately (`coordinateOrigin`). |
-| `COORDINATE_SYSTEM.CARTESIAN`         | [x, y, z] | identity units | A linear system with no interpretation for pure info-vis layers. Viewports can be used without supplying geospatial reference points. |
-| `COORDINATE_SYSTEM.LNGLAT_DEPRECATED`| [longitude, latitude, altitude] | meters | A lower precision version of the `COORDINATE_SYSTEM.LNGLAT` mode, that was the default until deck.gl v6.2. Will be removed in a future release. |
+Throughout this document you will find reference to the following **coordinate spaces** in deck.gl's projection system:
+
+##### World space
+
+The natural coordinate system of a dataset. It is usually determined by the source of the data, like the device it is generated on, and the format that it is stored as on disk or a cloud data warehouse.
+
+Some examples of different world space definitions include a [Shapefile](https://en.wikipedia.org/wiki/Shapefile) with any [standard geospatial reference](https://epsg.io/); WGS84 as used by GeoJSON or KML; GPS traces; LIDAR scans; info-vis values that do not map to real-world dimensions; [Slippy Map](https://wiki.openstreetmap.org/wiki/Slippy_Map) tiles from services like OpenStreetMaps and Mapbox; 3D tiling standards such as [Cesium 3D Tiles](https://www.ogc.org/standards/3DTiles) and [Esri I3S](https://www.ogc.org/standards/i3s).
+
+Positions in different world spaces come with different scales and orientations, may be measured relative to static or dynamic reference points, and in many cases are not linear. It is common for an application to overlay multiple datasets from different world spaces into the same 3D view.
+
+##### Common space
+
+To correctly compose data from various world spaces together, deck.gl transforms them into common space - an unified, intermediate 3D space that is a [right-handed Cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system#In_three_dimensions). Once positions are in the common space, it is safe to add, substract, rotate, scale and extrude them as 3D vectors using standard linear algebra. This is the basis of all geometry processing in deck.gl layers.
+
+The transformation between the world space and the common space is referred to in deck.gl documentation as "project" and "unproject", a process controlled by both the specification of the world space, such as WGS84, and the [projection mode](/docs/developer-guide/views.md), such as Web Mercator. Projections are implemented as part of deck.gl's core.
```

Comment:
```suggestion
The transformation between the world space and the common space is referred to in deck.gl documentation as "project" (world space to common space) and "unproject" (common space to world space), a process controlled by both the specification of the world space, such as WGS84, and the [projection mode](/docs/developer-guide/views.md), such as Web Mercator. Projections are implemented as part of deck.gl's core.
```
Not sure if it's helpful here to be more explicit about project/unproject. I always forget which is which.